From 0b8c70460c00b519852d04a4d8e28fcc82f70f71 Mon Sep 17 00:00:00 2001
From: Jason King <jason.king@joyent.com>
Date: Tue, 9 Apr 2019 19:09:37 +0000
Subject: [PATCH] OS-7723 libdemangle missing dependency on libcustr Reviewed
 by: Dan McDonald <danmcd@joyent.com> Reviewed by: Cody Peter Mello
 <cody.mello@joyent.com> Approved by: Cody Peter Mello <cody.mello@joyent.com>

---
 usr/src/lib/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/usr/src/lib/Makefile b/usr/src/lib/Makefile
index 669cffa0d3..189f846829 100644
--- a/usr/src/lib/Makefile
+++ b/usr/src/lib/Makefile
@@ -615,6 +615,7 @@ libcmdutils:	libavl
 libcpc:		libpctx
 libcrypt:	libgen
 libctf:		libdwarf
+libdemangle:	libcustr
 libdevid:	libdevinfo
 libdevinfo:	libsec libgen
 libdhcpagent:	libdhcputil libuuid libdlpi libcontract
-- 
2.21.0

